packages/webpack/src/config/base.js

```diff
@@ -113,8 +115,11 @@ export default class WebpackBaseConfig {
         }
       )
     }
-
-    const defaultPreset = [require.resolve('@nuxt/babel-preset-app'), {}]
+    const defaultPreset = [require.resolve('@nuxt/babel-preset-app'), {
+      corejs: {
+        version: corejs === 'auto' ? createRequire(rootDir)('core-js').version[0] : corejs
```

Comment:
What do you think to compute `version` outside (L118) and also do a validation to ensure it is either `"2"` or `"3"`?

Reply:
What we do on invalid values? choose default value or raise an error?

Reply:
A warn could be enough :)

https://babeljs.io/docs/en/babel-preset-env#corejs

Added here: https://github.com/nuxt/nuxt.js/pull/7785/commits/4424e9c61556b5005f6955d619ddc99c00349ba5#diff-d8071aa99dfe30c3c1c6135dfd903334R129

Reply:
👍 ❤️